src/test/java/uk/gov/register/util/RegisterComparerTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.register.util;
+
+import org.junit.Test;
+import uk.gov.register.core.RegisterMetadata;
+import uk.gov.register.core.RegisterName;
+
+import java.util.Arrays;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class RegisterComparerTest {
+    @Test
+    public void equals_shouldReturnFalseIfPhaseDiffers() {
+        RegisterMetadata one = new RegisterMetadata(
+                new RegisterName("Hello"), Arrays.asList(), "foo", "bar", "baz", "beta");
```

Comment:
Could we maybe change the values to something more self describing, for example:
`new RegisterName("country"), Arrays.asList(), "Copyright", "FCO", "Register of countries", "beta")` ?